Gemfile

```diff
@@ -1,8 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.3.5'
 
-gem 'test-kitchen'
 gem 'kitchen-docker'
 gem 'kitchen-salt'
 gem 'kitchen-inspec'
-gem 'rake'
```

Comment:
Could you supply the version numbers as well?  We've had some trouble with this otherwise:

```Gemfile
source "https://rubygems.org"

gem 'kitchen-docker', '>= 2.9'
gem 'kitchen-salt', '>= 0.6.0'
gem 'kitchen-inspec', '>= 1.1'
```